changestopiglatin.txt

```diff
@@ -0,0 +1,13 @@
+
+pyg = 'ay'
+
+original = raw_input('Enter a word:')
+
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
+    print new_word
+else:
+    print 'empty try again'
```

Comment:
this prints `empty try again` even when input was given, but it was not alphanumeric.